package.json

```diff
@@ -34,26 +34,22 @@
     "perf:meta": "ab -t 5 -c 20 -T application/json http://localhost:5555/tables",
     "generate:types": "supabase gen types typescript --local > ./supabase/functions/common/database-types.ts"
   },
-  "overrides": {
-    "@code-hike/mdx": {
-      "@code-hike/lighter": "^0.6.7"
-    }
-  },
   "devDependencies": {
-    "eslint": "^8.41.0",
+    "eslint": "^8.45.0",
     "eslint-config-custom": "*",
     "prettier": "^2.8.8",
     "prettier-plugin-sql-cst": "^0.5.0",
+    "sass": "^1.68.0",
```

Comment:
random package added?

Reply:
this is a weird one to add in the root

Reply:
Next requires the sass package when using `.scss` files

Reply:
but why in the root ?

Reply:
nvm. just pointing it out

Reply:
good point – I will move from root in a follow up PR